templates/clientgo/types.go.tmpl

```diff
@@ -0,0 +1,26 @@
+package {{.PackageName}}
+
+{{range .Enums}}
+type {{.Name}} string
+{{$typeName := .Name}}
+
+var {{.Name}}Types = struct {
+  {{- range .Values}}
+	/* {{ .Description }}*/
```

Comment:
Is this comment style a Go-specific thing for displaying documentation on godoc.org? If not, any thoughts on standardizing on `//` for comments and godoc things?

Reply:
The `/* */` is a multi-line comment for go.  I think we need a method to take the multi-line doc string and convert it to the `//` style.  We have some filtering methods currently, but nothing I saw to transform the string.  Perhaps the `GetDescription()` could do this for us.

Reply:
This should be resolved for cleaner description handling now.